Frends.Salesforce.CreateSObject/Frends.Salesforce.CreateSObject/Definitions/Options.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using static Frends.Salesforce.CreateSObject.Definitions.Enums;
+
+namespace Frends.Salesforce.CreateSObject
```

Comment:
Please use one-line namespace definition. Also, namespace should be `Frends.Salesforce.CreateSObject.Definitions`.